### DISCUSSION TITLE
First jam on the Hacktribe

### DISCUSSION THREAD

**Discussion by djokenzi** [bangcorrupt/hacktribe#175] [Show and tell]
Here's my first jam on the Hacktribe. I've added a link to the hacktribe github under the video to spread the word about this awesome mod. 

https://youtu.be/aVMGLA9Lavc

Apart from sharing this so you guys can check it out, I'd also like to ask the community about something. If you listen carefully to the hi-hat pattern, you'll notice a strange behaviour - it's not exactly following the grid (some might even say that's a good thing :)). This wasn't intentional, though, and it looks like CPU is struggling to keep up with all the elements of the track.

My question is: Has anyone else experienced similar issues on the Hacktribe, and more importantly, do they tend to happen on the stock firmware as well? I have zero experience with the sampler version of the Electribe - I literally bought a used unit a couple of days ago and immediately flashed it with the hacktribe firmware. But I own a synth version for a couple of years now and I never had such issues.

**Comment by bangcorrupt** [comment]
Could be related to #71.

Does it get worse if you scroll through FX types or samples quickly?

**Reply by djokenzi** [reply]
Could be. I'll try that out as soon as I get the chance.

**Reply by djokenzi** [reply]
It definitely does get worse, especially when I'm scrolling through samples. Hope this can be fixed in future.

**Reply by bangcorrupt** [reply]
If you're running the latest version of Hacktribe this should be fixed.  Performance should be  the same as the factory sampler firmware.

**Reply by djokenzi** [reply]
Thanks bangcorrupt, I'll try to flash the latest version soon.

**Comment by fanko89** [comment]
This is so awesome! Sadly, I have a bunch of projects already saved on my electribe sampler and the hack makes it so none of the previous sound banks line up. I think I’ll stick to the original, but I love what you are doing; the new synths are sick!

**Reply by bangcorrupt** [reply]
#142

**Comment by djokenzi** [comment]
I fired up my Hacktribe today, after a long time, and played around with it. I didn't use it much since I bought the Akai MPC Live. Anyway, here are the results: 

https://youtu.be/hdKxmmR24UQ

Hope you like it! :)

**Reply by BKLronin** [reply]
Good stuff, the deep house one especially.
